sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -23,7 +23,7 @@
 #include <unordered_set>
 #include <vector>
 
-#ifdef TESTING_BUILD
+#if defined(TESTING_BUILD)
```

Comment:
Did you run some regex or find/replace to catch all these? Are we sure we updated all of them?

Reply:
Yes, 100% possitive :)